bridge/test/nftbridge/NFTBridge_test.js

```diff
@@ -767,6 +813,52 @@ contract("Bridge NFT", async function(accounts) {
             await assertTokenHasBeenClaimed(this.sideNFTBridge.address, anotherAccount, sideToken)
           });
 
+      it("should claim with the fallback function", async function () {
+        await assertTokenIsLockedInBridge(this.NFTBridge.address, anotherAccount, this.token);
+        const receipt = await this.sideNFTBridge.claimFallback(
+          {
+            to: anotherAccount, from: anAccount, tokenId: tokenId, tokenAddress: tokenAddress,
+            blockHash: blockHash, transactionHash: transactionHash, logIndex: logIndex
+          },
+          {from: anAccount}
+        );
+
+        truffleAssert.eventEmitted(receipt, claimedNFTTokenEventType, (event) => {
+          return (
+            event._transactionHash === transactionHash &&
+            event._originalTokenAddress === tokenAddress &&
+            event._to === anotherAccount &&
+            event._sender === anAccount &&
+            event._tokenId.eq(new BN(tokenId)) &&
+            event._blockHash === blockHash &&
+            event._logIndex.eq(new BN(logIndex)) &&
+            event._receiver === anAccount
+          );
+        });
+      });
+
+      it("should check if it was claimed without claiming", async function () {

```

Comment:
I'd rename to ""should check if it was claimed without claiming when calling hasBeenClaimed"

Reply:
done at https://github.com/rsksmart/tokenbridge/pull/268/commits/9c4e1c7b64b7379ff804bb3f768eed9c96a7461b